File: Arrays/04_Sort an array of 0s, 1s and 2s/Solution.cpp
/**
 *  Name - Jatin Sharma
 *  Date - Oct, 27, 2020
 *  Link - https://practice.geeksforgeeks.org/problems/sort-an-array-of-0s-1s-and-2s/0
**/

#include <iostream>
using namespace std;

int main()
{
    int testCases, size;
    cin >> testCases;

    while (testCases--)
    {
        cin >> size;
        int arr[size];
        for (int i = 0; i < size; i++)
        {
            cin >> arr[i];
        }

        int temp = 0;
        for (int i = 0; i < size; i++)
        {
            int min = arr[i];
            for (int j = i; j < size; j++)
            {
                if (min > arr[j])
                {
                    temp = min;
                    min = arr[j];
                    arr[j] = temp;
                    j--;
                }
            }
            cout << min << " ";
        }
        cout << endl;
    }
    return 0;
}

File: Arrays/04_Sort an array of 0s, 1s and 2s/Solution.py
'''
 *  Name - Jatin Sharma
 *  Date - Oct, 27, 2020
 *  Link - https://practice.geeksforgeeks.org/problems/sort-an-array-of-0s-1s-and-2s/0
'''

testCases = int(input())

while testCases>0:
    size = int(input())
    arr = list(map(int, input().split()))

    for i in range(0, size):
        for j in range(i+1, size):
            if arr[i] > arr[j]:
                arr[i], arr[j] = arr[j], arr[i]

    print(*arr)
    testCases -=1